src/app/page.tsx

```diff
@@ -1,36 +1,35 @@
 import Link from "next/link";
 
-import { LatestPost } from "~/app/_components/post";
-import { getServerAuthSession } from "~/server/auth";
 import { api, HydrateClient } from "~/trpc/server";
-import { UserProfile } from "./_components/userProfile/UserProfile";
+import { getServerAuthSession } from "~/server/auth";
+import UserProfile from "~/app/_components/userProfile/UserProfile";
 
 export default async function Home() {
-  const hello = await api.post.hello({ text: "from tRPC" });
-  const session = await getServerAuthSession();
-
-  void api.post.getLatest.prefetch();
+  const SESSION = await getServerAuthSession();
```

Comment:
I don't think all const variables need to be upper case, especially on FE. Reserver upper case for const variables that aren't dynamic